docs/index.md

```diff
@@ -7,305 +7,46 @@ page_keywords: docker, swarm, clustering
 # Docker Swarm
 
 Docker Swarm is native clustering for Docker. It turns a pool of Docker hosts
-into a single, virtual host.
+into a single, virtual host. Because Docker Swarm serves the standard Docker
+API, any tool that already communicates with a Docker daemon can use Swarm to
+transparently scale to multiple hosts. Supported tools include, but are not  limited to, the following: 
```

Comment:
How about getting rid of those examples? I'm not even sure they work beside `Docker Compose`. /cc @vieux 

How about:

> transparently scale to multiple hosts, including the Docker CLI and Docker Compose.